Directory.Build.props

```diff
@@ -88,21 +88,25 @@
     <MicrosoftGraphBetaVersion>4.57.0-preview</MicrosoftGraphBetaVersion>
     <MicrosoftExtensionsHttpVersion>3.1.3</MicrosoftExtensionsHttpVersion>
     <MicrosoftIdentityAbstractions>6.0.0</MicrosoftIdentityAbstractions>
-    <NetNineRuntimeVersion> 9.0.0-preview.4.24266.19</NetNineRuntimeVersion>
-    <AspNetCoreNineRuntimeVersion> 9.0.0-preview.4.24267.6</AspNetCoreNineRuntimeVersion>
+    <NetNineRuntimeVersion>9.0.0-preview.6.24327.7</NetNineRuntimeVersion>
+    <AspNetCoreNineRuntimeVersion>9.0.0-preview.6.24328.4</AspNetCoreNineRuntimeVersion>
+    <!--CVE-2024-30105-->
+    <SystemTextJsonVersion>8.0.4</SystemTextJsonVersion>
```

Comment:
seems like this should have been the fixed version of the oldest currently support .net version which is .NET 6.  Then in .net 8 and higher specific fields set the higher versions.

With this going to pull in .NET 8 oriented libraries in .NET 6 targeted applicaitons

Reply:
It's not going to pull .NET 8 libraries in .NET 6 applications as the libraries have different target framework